common/src/main/java/com/wynntils/wynn/model/questbook/QuestBookManager.java

```diff
@@ -309,6 +309,18 @@ public static List<QuestInfo> getQuests() {
     }
 
     public static List<QuestInfo> getQuestsSorted(QuestSortOrder sortOrder) {
+        return sortQuestInfoList(sortOrder, quests);
+    }
+
+    public static List<QuestInfo> getMiniQuests() {
+        return miniQuests;
+    }
+
+    public static List<QuestInfo> getMiniQuestsSorted(QuestSortOrder sortOrder) {
+        return sortQuestInfoList(sortOrder, miniQuests);
+    }
+
+    private static List<QuestInfo> sortQuestInfoList(QuestSortOrder sortOrder, List<QuestInfo> questInfoList) {
```

Comment:
This will work much better for miniquests if you replace `quests` by `questInfoList` in the switch below. ;-) (AKA: ooops :)) 

(Also maybe just `questList`?)

Reply:
I fixed it for you. :)